HesoyamHospital/Backend/Service/AppointmentMapper.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Backend.Model.PatientModel;
+using Backend.Model.UserModel;
+using Backend.Util;
+
+namespace Backend.Service
+{
+    public static class AppointmentMapper
+    {
+        public const long AppointmentDurationMinutes = 30;
+        public static Appointment AppointmentDtoToAppointment(AppointmentDTO dto)
+        {
+            Appointment appointment = new Appointment();
```

Comment:
Try using a constructor here instead.

Reply:
Thanks for suggestion, we solved this problem.